src/docs/v10-vs-v11-retrospective.md

```diff
@@ -0,0 +1,643 @@
+# Merging the Berkeley schema into the NMDC schema
+
+This document is a retrospective on the changes made to the NMDC schema between major versions 10 and 11. The
+development of version 11 was launched during a meeting at the Lawrence Berkeley National Laboratory in December 2023,
+and the development took place in a fork of the nmdc-schema called berkeley-schema-fy24. This is the first time a major
+update had comprehensive planning and development in a fork. A subsequent update in the same style has not planned yet
+but is not ruled out.
+
+Motivations for the Berkeley schema development included:
+
+- First class representations of "omics" methods beyond nucleic acid analysis, like mass-spectrometry based
```

Comment:
```suggestion
- First class representations of "omics" methods beyond nucleic acid analysis, like mass spectrometry-based
```

I'm used to hyphenating "based" with the word that precedes it, when the combination is being used as an adjective describing the noun that follows "based." I don't know the name of the grammatical rules that apply here.


Here's a related Q/A on StackExchange:

https://english.stackexchange.com/questions/207273/use-of-a-hyphen-with-the-word-based